delta/delta_1.4.1_swmm_catchment_areas.sql

```diff
@@ -0,0 +1,231 @@
+DROP VIEW IF EXISTS qgep_swmm.vw_subcatchments CASCADE;
+CREATE OR REPLACE VIEW qgep_swmm.vw_subcatchments AS
```

Comment:
@3nids should all views be (re-created) in pre- / post-all?

Reply:
@m-kuhn yes. but this should be done in
https://github.com/QGEP/datamodel/blob/master/view/create_views.py#L74
https://github.com/QGEP/datamodel/blob/master/view/drop_views.sql

Reply:
CC @tproduit

Reply:
I have added the swmm view in these two files